packages/clarity-js/src/layout/index.ts

```diff
@@ -7,17 +7,17 @@ import * as region from "@src/layout/region";
 
 export function start(): void {
     doc.start();
+    region.start();
     dom.start();
     mutation.start();
     discover.start();
     box.start();
-    region.reset();
 }
 
 export function stop(): void {
+    region.stop();
```

Comment:
Why is order of these in start/stop different and is that intentional? If intentional, could you add a quick comment why?

Reply:
It is based on the dependency order. I will add comment.